logzio-log-sender/src/test/java/io/logz/BaseSenderTest.java

```diff
@@ -0,0 +1,120 @@
+package io.logz;
+
+import org.assertj.core.api.Assertions;
+import org.junit.After;
+import org.junit.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.net.BindException;
+import java.net.ServerSocket;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public abstract class BaseSenderTest {
```

Comment:
I think we can avoid creating base classes just as this and the TestLogzioSender (Let's check that name as well). We can create a suite of tests, and create a Sl4j Appender: once wrapping LogzioSender for testing the sender, and once using Logback to test this FW functionality.